Symfony/CS/FixerFactory.php

```diff
@@ -76,6 +83,20 @@ public function getFixers()
     }
 
     /**
+     * Check if fixer exists.
+     *
+     * @param string $name The rule name of the fixer
+     *
+     * @return bool
+     */
+    public function hasFixer($name)
```

Comment:
Look on interface - input is name of rule, output is bool.
What is done under the hood (like checking fixers) isn't crucial for interface